### ISSUE TITLE
Lineage graph and list fail with versioned models.

### ISSUE THREAD

**Issue by Kamal-Klm** [AltimateAI/vscode-dbt-power-user#589]
### Expected behavior

Selecting a model that has multiple versions should show the lineage connection of the version which dbt's`ref` macro resolves. 
If `ref` yields v1, then lineage and the model's hierarchy list should render parents & children specific to model.v1.

### Actual behavior

Selecting a versioned model renders an empty lineage graph and an empty hierarchy parent/child list for that model.
Basically power user might fail to parse manifest.json for this node.

### Steps To Reproduce

Select any versioned model and select the lineage graph or open the parent/child list.

### Operating System

macOS

### dbt version

1.6

### dbt Adapter

postgres

### dbt Power User version

v0.17.7

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

**Comment by mdesmet** [comment]
Duplicate of https://github.com/AltimateAI/vscode-dbt-power-user/issues/872
